src/main/java/games/strategy/engine/framework/GameDataFileUtils.java

```diff
@@ -0,0 +1,91 @@
+package games.strategy.engine.framework;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.apache.commons.io.IOCase;
+
+import com.google.common.annotations.VisibleForTesting;
+
+/**
+ * A collection of utilities for working with game data files.
+ */
+public final class GameDataFileUtils {
+  private static final String EXTENSION = ".tsvg";
+
+  private static final String LEGACY_EXTENSION = ".svg";
+
+  /**
+   * Macs download a game data file as "tsvg.gz", so that extension must be used when evaluating candidate game data
+   * files.
+   */
+  private static final String MAC_OS_ALTERNATIVE_EXTENSION = "tsvg.gz";
```

Comment:
Not sure why the original code didn't include a leading period, but I left it unchanged because I have no evidence to question the original author's intent and can't test this on Mac.